### ISSUE TITLE
[bug] explain reports completed successful run as running

### ISSUE THREAD

**Issue by Luffy-nani** [rajudandigam/agent-inspect#173]
### Summary

The explain command reports a run's status as "running" even after the run has completed successfully, while list and view both correctly show "success" for that same run.

### Area

CLI (list/view/search/report/diff/check and friends)

### AgentInspect version

6.7.2

### Steps to reproduce

Run the following synthetic script:

```js
import { inspectRun, step } from "agent-inspect";
const delay = (ms) => new Promise((r) => setTimeout(r, ms));

await inspectRun("support-agent", async () => {
  await step("plan", async () => { await delay(40); return { intent: "refund-policy" }; });
  const policy = await step.tool("retrieve-policy", async () => { await delay(60); return { text: "synthetic policy text" }; });
  return step.llm("generate-answer", async () => { await delay(80); return `Policy: ${policy.text}`; });
}, { traceDir: "./.agent-inspect" });
```

Then run:

```bash
node demo.mjs
npx agent-inspect list --dir ./.agent-inspect      # confirm status: success
npx agent-inspect explain <runId> --dir ./.agent-inspect
```

### Expected behavior

explain should report run.status: "success", matching what list/view show for the same run.

### Actual behavior

explain reports run.status: "running" for a run that has already completed successfully.

### Environment

Node v22.22.2, Linux (Ubuntu 24), npm

### Privacy confirmation

- [x] This report contains synthetic or redacted data only — no real prompts, production logs, tokens, or PII.
- [x] This is not a request for hosted observability, vendor upload, replay, or cost analytics (see docs/LIMITATIONS.md).

**Comment by abhay-codes07** [comment]
Root cause: the run tree that `explain` reads through `openTrace` derives its status in `TreeBuilder.computeRunStatus`, which returned "running" if any event carried a "running" status. The v0.1 to persisted bridge writes each node as a started event (status "running") followed by a completed event, so a finished run still contains "running" started rows and the whole run gets reported as running. `list`/`view` go through `extractMetadata`/`buildRunSummary`, which read the run completion status, so they correctly show success.

Fix derives the run status from the RUN node lifecycle, matching `manualTraceEventsToRunTree` and `extractMetadata`: the run is running only until its RUN completion is present, then it takes that terminal status. Traces with no RUN node keep the old fallback. I also normalized the explain status output to the success/error/running vocabulary so it matches list/view for the same run.

Verified against the repro: `list` shows success, `explain` now reports success in human output, `--json`, and the `run.status` fact. PR incoming.
